recipe-server/client/control_new/routes.js

```diff
@@ -68,20 +68,25 @@ export const {
   basename: '/control-new',
 });
 
-export function resolveRoutes({ router }) {
-  let content = <MissingPage />;
+@connect(state => ({
+  router: state.router,
+}))
+export default class Router extends React.Component {
+  static propTypes = {
+    router: PropTypes.object.isRequired,
+  };
 
-  if (router.route) {
-    content = <router.result.component />;
-  }
+  render() {
+    const { router } = this.props;
 
-  return <App>{content}</App>;
+    return (
```

Comment:
IMO, I kinda liked this the old way - feels cleaner to do `<App>{content}</App>`. What if:
```
const content = router.route ? <router.result.component /> : <MissingPage />;
return <App>{content}</App>;
```
?